core/vm/contracts.go

```diff
@@ -734,6 +734,11 @@ func (c *bls12381G1Mul) Run(input []byte) ([]byte, error) {
 	if p0, err = decodePointG1(input[:128]); err != nil {
 		return nil, err
 	}
+	// 'point is on curve' check already done,
+	// Here we need to apply subgroup checks.
```

Comment:
Please add a comment to `bls12381G1Add`.Run` to the effect "subgroup checks are omitted in G1 add, in accordance with spec" so we don't "discover" it two years from now and get a heart attack

Reply:
Same for g2 add